lippe

```diff
@@ -9,3 +9,4 @@ networks:
     - 10.13.128.0/20
   ipv6:
     - 2a03:2260:2004::/48
+    - 2001:bf7:1340::/44
```

Comment:
The whois for this network does not show a subassignmentto Freifunk Lippe, is there any documentation to confirm this change?